windows/test_msi.ps1

```diff
@@ -3,43 +3,60 @@ param (
     [string]$ARCH = "amd64",
     [string]$TAG = "v0.0.0",
     [string]$UPGRADE = "false", # upgrade: upgrade msi from last released version.
+    [string]$PKG_TYPE = "msi",
```

Comment:
you could use the ValidateSet here https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_functions_advanced_parameters?view=powershell-7.1#validateset-attribute

Reply:
don't need to change it now

Reply:
This looks great! I suck at powershell.